Add Discussions to CONTRIBUTING (#3613)
Alamofire/Alamofire: CONTRIBUTING.md

## Submitting Pull Requests

> You can contribute by fixing bugs or adding new features. For larger code changes, we first recommend discussing them in our [GitHub issues](https://github.com/Alamofire/Alamofire/issues) or on our [Swift Forums](https://forums.swift.org/c/related-projects/alamofire/). When submitting a pull request, please add relevant tests and ensure your changes don't break any existing tests (see [Automated Tests](#automated-tests) below).

Reviewer: ```suggestion
You can contribute by fixing bugs or adding new features. For larger code changes, we recommend first discussing your ideas on our [GitHub Discussions](https://github.com/Alamofire/Alamofire/discussions) or [Swift Forums](https://forums.swift.org/c/related-projects/alamofire/). When submitting a pull request, please add relevant tests and ensure your changes don't break any existing tests (see [Automated Tests](#automated-tests) below).
```

Reviewer: Would you mind accepting the switch to GitHub discussions as well?
